inject.js

```diff
@@ -825,6 +827,32 @@ function runAction(action, value, e) {
   log("runAction End", 5);
 }
 
+function injectScriptForSite() {
+  let inject = true;
+  const elt = document.createElement("script");
+
+  switch (true) {
+    case location.hostname == "www.netflix.com":
+      elt.src= chrome.runtime.getURL('scriptforsite/netflix.js');
+      break;
+    default:
+      inject = false;
+  }
+  if (inject) {
```

Comment:
Can we simplify it to perform a check on script.src?

Reply:
Sure, I've already submitted a new commit.